### PR TITLE
[IMP] Add Wemos D1 mini lite (ESP8285) board support

### PR DESCRIPTION
**Pull request by zyphos**
Tested with original Wemos D1 mini lite v1.0.0

### CODE REVIEW

File: boards.txt [hunk 1]
```diff
@@ -1319,6 +1319,111 @@ d1_mini.menu.FlashSize.4M1M.build.spiffs_blocksize=8192
...
+d1_mini_lite.upload.maximum_data_size=81920
+d1_mini_lite.upload.wait_for_upload_port=true
+d1_mini_lite.serial.disableDTR=true
+d1_mini_lite.serial.disableRTS=true
+
+d1_mini_lite.build.mcu=esp8266
+d1_mini_lite.build.f_cpu=80000000L
+d1_mini_lite.build.board=ESP8266_ESP01
+d1_mini_lite.build.core=esp8266
+d1_mini_lite.build.variant=generic
```
Comment (igrr) [inline boards.txt]: Should this be different from `d1_mini` board? Isn't the pinout the same?

File: boards.txt [hunk 2]
```diff
@@ -1319,6 +1319,111 @@ d1_mini.menu.FlashSize.4M1M.build.spiffs_blocksize=8192
...
+d1_mini_lite.upload.resetmethod=nodemcu
+d1_mini_lite.upload.maximum_size=1044464
+d1_mini_lite.upload.maximum_data_size=81920
+d1_mini_lite.upload.wait_for_upload_port=true
+d1_mini_lite.serial.disableDTR=true
+d1_mini_lite.serial.disableRTS=true
+
+d1_mini_lite.build.mcu=esp8266
+d1_mini_lite.build.f_cpu=80000000L
+d1_mini_lite.build.board=ESP8266_ESP01
```
Comment (igrr) [inline boards.txt]: This also needs to be updated (see `d1_mini` board)

**Comment by zyphos** [comment]
You are right, I updated the code.
Thank you.

**Comment by capedra** [comment]
@zyphos @igrr 

The updated code is wrong. The only way to make the Wemos D1 Mini Lite 1.0.0 is to set the **flash_mode** accordingly:
```
d1_mini_lite.build.flash_mode=dout
```
Attention: if you don't change the above line, the board will **NEVER** work.

**Comment by d-a-v** [comment]
I can confirm this is true. Currently this PR uses 
```
d1_mini_lite.build.flash_mode=dio
```
It flashes OK but does not boot properly.
However, changing ```dio``` to ```dout``` makes it work properly. 
@zyphos I wonder if you could make your board boot properly with ```dio``` ?
My board is ```D1 mini Lite v1.0.0```.

**Comment by d-a-v** [comment]
d1 mini lite is now in master's board.txt (dout).

File: boards.txt [hunk 3]
```diff
@@ -1318,6 +1318,111 @@ d1_mini.menu.FlashSize.4M1M.build.spiffs_end=0x3FB000
...
```
Comment (spencerkraisler) [inline boards.txt]: This isn't working for me. I replaced the boards.txt file with this file yet "Wemos D1 mini lite (ESP8285)" doesn't pop up in my boards on the Arduino IDE.

Comment (d-a-v) [reply boards.txt]: It is merged. Have a try with master. Restart the IDE after updating.

**Comment by d-a-v** [comment]
d1 mini lite is there. Closing.

**Pull request closed without merging**